edit-post/components/header/more-menu/index.js

```diff
@@ -38,6 +39,7 @@ const MoreMenu = () => (
 					filterName="editPost.MoreMenu.tools"
 				>
 					<TipsToggle onToggle={ onClose } />
+					<PublishSidebarToggle onToggle={ onClose } />
```

Comment:
I know it's unrelated to this PR, but the `onClose` method being used for `onToggle` several times and `onSelect` below makes me think it should be renamed to `closeMenu` or something. 🤷‍♂️